addons/nuxeo-ai-core/elements/actions/nuxeo-ai-bulk-add.js

```diff
@@ -0,0 +1,96 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { FiltersBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-filters-behavior.js';
+import { I18nBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-i18n-behavior.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '../nuxeo-ai-icons.js';
+
+/**
+ * `nuxeo-ai-bulk-add-button` an export progress template.
+ *
+ *     <nuxeo-ai-bulk-add-button provider=[[provider]]>
+ *     </nuxeo-ai-bulk-add-button>
+ *
+ * @memberof Nuxeo
+ */
+
+class BulkAddButton extends mixinBehaviors([I18nBehavior, FiltersBehavior], Nuxeo.Element) {
+  static get template() {
+    return html`
+      <nuxeo-operation-button
+        id="aiBulkAddOpBtn"
+        operation="Bulk.RunAction"
+        input="[[provider]]"
+        params="[[_params()]]"
```

Comment:
Just for the record, other possible approach for this would be a computed property.

Reply:
Just to confirm, it is expected to have this as params:
`{
      action: 'bulkEnrich',
      parameters: {},
  }`

If so, you may want to change the method itself.

Reply:
Since this method always returns the same object, you could even inline it:
```suggestion
        params='{ action: "bulkEnrich", parameters: "{}" }'
```